errdefs/errdefs.go

```diff
@@ -0,0 +1,24 @@
+package errdefs
+
+import "errors"
+
+type internalErr struct {
+	error
+}
+
+func (internalErr) System() {}
+
+type system interface {
+	System()
+}
+
+var _ system = internalErr{}
+
+func Internal(err error) error {
+	return internalErr{err}
+}
+
+func IsInternal(err error) bool {
```

Comment:
This matches the interface definition and behavior of https://github.com/containerd/errdefs/blob/124d0dc9f0cd27578004a3f14e01ef8a3456d147/errors.go#L304 (unreleased, not part of the v1).

In the future containerd `IsInternal()` function can be used instead in the caller side for equivalent behavior.